code/modules/mob/living/carbon/human/human_organs.dm

```diff
@@ -137,7 +137,7 @@
 		return
 
 	for (var/obj/item/organ/external/E in organs)
-		if(!E || !E.can_grasp || (E.status & ORGAN_SPLINTED))
+		if(!E || (E.limb_flags & ORGAN_CAN_GRASP) || (E.status & ORGAN_SPLINTED))
```

Comment:
Initial check reads: `IF NOT E.can_grasp`. New check reads: `IF E.limb_flags HAS FLAG ORGAN_CAN_GRASP`. Looks like you accidentally inverted it? Unless the `can_grasp` boolean was inverted already.

Reply:
fixed